content/en/account_management/guide/relevant-usage-migration.md

```diff
@@ -0,0 +1,509 @@
+- `rum`
+- `rum_replay`
+- `rum_lite`
+
+The SKUs that are not active for your org will be null. The 13 usage types will provide granular usage summaries under the RUM SKU.
+
+#### Current response structure
+
+A current example of a response for the [Get usage across your account][2] v1 endpoint follows:
+{{< code-block lang="json">}}
+{
+  "usage": {
+    "rum_session_count_agg_sum": 7441533,
+    "mobile_rum_session_count_flutter_agg_sum": 0,
+    "mobile_rum_session_count_ios_agg_sum": 0,
+    "rum_total_session_count_agg_sum": 7618504,
+    "rum_browser_and_mobile_session_count_agg_sum": 7441533,
+    "mobile_rum_session_count_android_agg_sum": 0,
+    "mobile_rum_session_count_reactnative_agg_sum": 0,
+    "mobile_rum_session_count_roku_agg_sum": 0,
+     },
+    { ... // Summary usage by sub-organization }
+}
+{{< /code-block >}}
+
+#### Upcoming response structure
+
+After October 1, 2024, the [Get usage across your account][2] v1 endpoint will have the following example structure:
+
+{{< highlight json "hl_lines=3 8 23" >}}
+{
+  "usage": {
+// Three keys representing SKUs that your RUM usage may be billed on 
+    "rum_total_session_count_agg_sum": null,
+    "rum_replay_session_count_agg_sum": 50,  
+    "rum_lite_session_count_agg_sum": null,
+
+// RUM usage types representing granular RUM usage data
+    "rum_browser_legacy_session_count_agg_sum": 0,
+    "rum_browser_lite_session_count_agg_sum": 183911,
+    "rum_browser_replay_session_count_agg_sum": 5576,
+    "rum_mobile_legacy_session_count_android_agg_sum": 0,
+    "rum_mobile_legacy_session_count_flutter_agg_sum": 0,
+    "rum_mobile_legacy_session_count_ios_agg_sum": 0,
+    "rum_mobile_legacy_session_count_reactnative_agg_sum": 0,
+    "rum_mobile_legacy_session_count_roku_agg_sum": 0,
+    "rum_mobile_lite_session_count_android_agg_sum": 0,
+    "rum_mobile_lite_session_count_flutter_agg_sum": 0,
+    "rum_mobile_lite_session_count_ios_agg_sum": 0,
+    "rum_mobile_lite_session_count_reactnative_agg_sum": 0,
+    "rum_mobile_lite_session_count_roku_agg_sum": 0,
+
+// Legacy usage keys are nulled
+   "rum_session_count_agg_sum": null,
+   "mobile_rum_session_count_flutter_agg_sum": null,
+   "mobile_rum_session_count_ios_agg_sum": null,
+   "rum_browser_and_mobile_session_count_agg_sum": null,
+   "mobile_rum_session_count_android_agg_sum": null,
+   "mobile_rum_session_count_reactnative_agg_sum": null,
+   "mobile_rum_session_count_roku_agg_sum": null
+
+  }
+},
+{… // Summary usage by sub-organization}
+{{< /highlight >}}
+
+### Indexed Logs
+
+Keys that represent total usage across all retentions will be deprecated and displayed as null. These keys are: 
+- `indexed_events_count_sum`
+- `live_indexed_events_agg_sum`
+- `rehydrated_indexed_events_agg_sum`
+
+The usage across all retention periods can continue be calculated by summing the individual per-retention keys. 
```

Comment:
I'm not sure what "individual per-retention keys" means, but this might be clear to users of the product

Reply:
@neko-dd @urseberry This means individual keys for each retention period since Indexed Logs are typically broken down by retention e.g. 1 day, 3 day, 90 day, 1 year retention periods. 

It might be more clear to say "The usage across all retention periods can continue to be calculated by summing the individual retention keys."